features/upload_cita_appointments.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Manual Upload of CITA appointments CSV
+  As a Pension Wise data analyst
+  I want to be able to manually upload the CITA appointments CSV
```

Comment:
I think this could be clearer as `I want to manually upload the CITA appointments CSV`.